packages/data-service/src/data-service.ts

```diff
@@ -380,7 +616,15 @@ class DataService extends EventEmitter {
     options: FindOptions,
     callback: Callback<Document[]>
   ): void {
-    this.client.find(ns, filter, options, callback);
+    // Workaround for https://jira.mongodb.org/browse/NODE-3173
```

Comment:
I think we can just drop this line? The NODE ticket is fixed, but I don’t think this is a workaround that we necessarily need to undo…

```suggestion
```

Reply:
Thanks for spotting, yeah that doesn't make any sense here, I've removed the remark!